examples/src/layers/examples.ts

```diff
@@ -0,0 +1,70 @@
+import type { ColumnRequest } from '~/common/loaders/scatterplot/scatterbrain-loader';
+import type { AnnotationGridConfig } from '~/data-sources/annotation/annotation-grid';
+import type { ZarrSliceConfig } from '~/data-sources/ome-zarr/planar-slice';
+import type { ZarrSliceGridConfig } from '~/data-sources/ome-zarr/slice-grid';
+import type { ScatterplotGridConfig, ScatterPlotGridSlideConfig } from '~/data-sources/scatterplot/dynamic-grid';
+
+const slide32 = 'MQ1B9QBZFIPXQO6PETJ';
```

Comment:
ok this whole file is unused - it was nice that it was pulled out of the code for the layers example logic, but the types got a bit messed up. 
either fix the types and use it, or delete it and keep the actual values on line 745 of src/layers.ts